pkg/plugin/plugin_test.go

```diff
@@ -210,7 +217,19 @@ var _ = Describe("CNI Plugin", func() {
 				"bridge": "%s"
 			}`, BRIDGE_NAME)
 			It("should successfully complete ADD and DEL commands", func() {
-				testAddDel(conf, false, false)
+				testAddDel(conf, false, false, "")
+			})
+		})
+		Context("with specific VLAN ID ranges set on port", func() {
+			conf := fmt.Sprintf(`{
+				"cniVersion": "0.3.1",
+				"name": "mynet",
+				"type": "ovs",
+				"bridge": "%s",
+				"trunk": [ {"minID": 10, "maxID": 12}, {"id": 15}, {"minID": 17, "maxID": 18}  ]
```

Comment:
Please add more tests especially to 'splitVlanIds' method with different inputs. For example - 
"trunk": [  {"id": 15},   {"id": 19},   {"id": 40} ]
"trunk": [ {"minID": 10, "maxID": 12},  {"minID": 19, "maxID": 20} ]
"trunk": [ {"minID": 10, "maxID": 14},  {"id": 11},  {"minID": 13, "maxID": 13} ]

As well as error inputs
"trunk": [  {"id": 15},   {"id": 15},   {"id": -20} ]
"trunk": [ {"minID": 10, "maxID": 12},  {"minID": 11, "maxID": 5} ]

Reply:
Done.